### PR TITLE
Feature/dragen cli

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### This PR fixes:

- A draft of a dragen rd dna pipeline

### How to test:

- Automatic and continuous test pass

### Expected outcome:
- Installation, unit and integration tests pass

### Review:
- [x] Code review
- [x] New code is executed and covered by tests
- [x] Tests pass

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,11 +7,6 @@ sudo: required
 before_install:
 ## Install SSL
   - sudo apt-get update
-#  - sudo apt-get install libssl1.0.0 libssl-dev libcrypto++9 libcrypto++-dev
-#  - cd /lib/x86_64-linux-gnu
-#  - sudo ln -s libssl.so.1.0.0 libssl.so.10
-#  - sudo ln -s libcrypto.so.1.0.0 libcrypto.so.10
-#  - cd -
```
Comment (jemten) [inline .travis.yml]: Nice to clean up the travis file a bit :building_construction:

File: definitions/dragen_rd_dna_parameters.yaml [hunk 2]
```diff
@@ -0,0 +1,812 @@
...
+infile_dirs:
+  associated_recipe:
+   - mip
+  exists_check: directory
+  data_type: HASH
+  type: path
+  update_path: absolute_path
+picardtools_path:
+  associated_recipe:
+#   - dragen_dna
```
Comment (jemten) [inline definitions/dragen_rd_dna_parameters.yaml]: remove?

File: definitions/dragen_rd_dna_parameters.yaml [hunk 3]
```diff
@@ -0,0 +1,812 @@
...
+  analysis_mode: case
+  associated_recipe:
+   - mip
+  data_type: SCALAR
+  default: 1
+  file_tag: _drg
+  outfile_suffix: ".vcf"
+#  program_executables:
+ #  - dragen
+  type: recipe
```
Comment (jemten) [inline definitions/dragen_rd_dna_parameters.yaml]: Remove? 
Or is this something that will be activated once dragen is actually installed?

File: definitions/rd_dna_dragen_parameters.yaml [hunk 4]
```diff
@@ -165,27 +165,13 @@ sv_vcfanno:
   data_type: SCALAR
   default: 0
   type: recipe_argument
```
Comment (jemten) [inline definitions/rd_dna_dragen_parameters.yaml]: What separates this file from dragen_rd_dna_parameters.yaml?

File: lib/MIP/Check/Pipeline.pm [hunk 5]
```diff
@@ -165,9 +416,9 @@ sub check_rd_dna {
     ## Checks parameter metafile exists and set build_file parameter
     check_parameter_metafiles(
         {
-            parameter_href        => $parameter_href,
             active_parameter_href => $active_parameter_href,
             file_info_href        => $file_info_href,
+            parameter_href        => $parameter_href,
```
Comment (jemten) [inline lib/MIP/Check/Pipeline.pm]: Good to see that there still are some parameters that can be ordered AB ;)

File: definitions/dragen_rd_dna_parameters.yaml (hunk 2 above)
Comment (henrikstranneheim) [reply definitions/dragen_rd_dna_parameters.yaml]: No I left these comments intentionally to be activated once we actually have a dragen to interact with.

File: lib/MIP/Check/Pipeline.pm (hunk 5 above)
Comment (henrikstranneheim) [reply lib/MIP/Check/Pipeline.pm]: 😄

File: definitions/rd_dna_dragen_parameters.yaml (hunk 4 above)
Comment (henrikstranneheim) [reply definitions/rd_dna_dragen_parameters.yaml]: I wonder how that got there! I will remove it

**Pull request merged**